modules/nginx/manifests/init.pp

```diff
@@ -100,6 +100,20 @@
     }
 
 
+    ##################
+    ## redirect map ##
+    ##################
+
+    file { "${nginx_conf_dir}/redirect_map.conf":
+        ensure  => present,
+        mode    => 0640,
+        owner   => $owner,
+        group   => $group,
+        source  => "puppet:///modules/nginx/redirect_map.conf",
+        require => $nginx_config_requires,
+    }
```

Comment:
There's a potential bootstrapping issue here in that the service could be installed and started before the redirect_map.conf file is placed. I think that nginx service should have this file added to its `require` dependency list.